exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -45,10 +66,33 @@ type prwExporter struct {
 	retrySettings     configretry.BackOffConfig
 	wal               *prweWAL
 	exporterSettings  prometheusremotewrite.Settings
+	telemetry         prwTelemetry
+}
+
+func newPRWTelemetry(set exporter.CreateSettings) prwTelemetry {
+
+	meter := metadata.Meter(set.TelemetrySettings)
+	// TODO: create helper functions similar to the processor helper: BuildCustomMetricName
+	prefix := "exporter/" + metadata.Type + "/"
+
+	failedTranslations, _ := meter.Int64Counter(prefix+"failed_translations",
```

Comment:
Don't ignore instrument creation errors.  These should be propagated up the call stack if they can't be handled here.

Reply:
I don't know why I thought it was ok to do this. fixed.